flybywiresim/a32nx
# refactor: rename package from A32NX to flybywire-aircraft-a32nx

Fixes #[issue_no]

## Summary of Changes

This PR changes the package name from A32NX to flybywire-aircraft-a32nx. The reason for this is the old package name was made before we knew of the proper naming conventions.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

There are no new features, and nothing was removed. This is just a simple rename. Make sure nothing is broken and everything behaves the way that it should.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **flybywire-aircraft-a32nx** link
